package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "description": "Use webpack with karma and fast, file source maps",
   "peerDependencies": {
-    "webpack": "^1.4.0 || ^2.1.0-beta"
+    "webpack": "^1.4.0 || ^2 || ^2.1.0-beta || ^2.2.0-rc"
```

Comment:
Yes, have tried against both 2.2 and latest 2.3 rc and no issues. I have changed`^2` to`^2.0.0`, both mean `>=1.0.0 <2.0.0` range.